cypress/integration/chi-vue/data-table.spec.js

```diff
@@ -338,12 +331,8 @@ describe('Data Table', () => {
       cy.get(
         `[data-cy='data-table-truncation'] .${DATA_TABLE_CLASSES.HEAD} .${DATA_TABLE_CLASSES.CELL}`
       )
-        .children()
```

Comment:
The test has been changed as Head cell markup is updated with the class `-text--truncate` set another div